src/registrar/views/domain.py

```diff
@@ -142,6 +155,13 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+class DomainDNSView(DomainPermissionView):
+
```

Comment:
**nitpick** eye twitch here too at the added line. feel free to ignore this comment